ubi_manifest/worker/tasks/content_audit.py

```diff
@@ -70,7 +76,9 @@ def content_audit_task() -> None:
             in_repos = client.search_repository(
                 Criteria.with_id(out_repo.population_sources)
             )
-            in_mod_rpm_filenames = get_pkgs_from_all_modules(in_repos)
+            modular_rpm_filenames = f_proxy(
+                executor.submit(get_pkgs_from_all_modules, in_repos)
```

Comment:
I guess you need to query for modular rpms also in `out_repo` and combine these with modular rpms from `in_repos`. Or maybe better solution would to remove modular rpms from `out_rpms_result`.

Reply:
@rbikar, modular RPMs from out_repo are filtered in the content check but based on in_repo modular filenames. So, while the filenames **_should_** cover the out_rpms as well, maybe it would be better to add out_repo modular RPM filenames to modular_rpm_filenames just to be sure.

Reply:
I've added the out_repo to the list of repos from which modular filenames are derived in addition to the existing check of out_rpms.